content/news/2023-10-06-COURTTOOL.md

```diff
@@ -0,0 +1,614 @@
++++
+aliases = ["/court-tool/", "/courttool/"]
+authors = ["Elizabeth Estrada"]
+blurb = "From abortion to the death penalty to education, Pennsylvania judges regularly shape, define, and affect policies on hot-button topics."
+byline = "Elizabeth Estrada"
+extended-kicker = "2023"
+image = "2023/10/01jh-nnsg-6zn1-3c7b.jpeg"
+image-credit = "Daniel Fishel / for Spotlight PA"
+image-description = "Gavel with election topics in the background"
+kicker = "Elections"
+layout = "custom-court-tool"
+published = 2023-10-06T05:00:00-04:00
+slug = "pennsylvania-judges-supreme-superior-commonwealth-court-interactive-tool"
+title = "INTERACTIVE TOOL: See how electing judges affects you and the issues you care about most"
+title-tag = "Discover how PA judges shape policies on abortion, education and more with this interactive tool."
+description = "Use this interactive tool to discover how Pennsylvania judges shape policies on abortion, education, and more."
+topics = ["Elections"]
+series = []
+credits = """
```

Comment:
There's not a way to set arbitrary params in Almanack. That said, Almanack should preserve a copy of parameters it doesn't understand when it reads a file in GitHub.